### PR TITLE
PS remoting support

### PR DESCRIPTION
**Pull request by mossbeard**

### CODE REVIEW

**Comment by mossbeard** [comment]
RM supported

File: src/QlikConnectorPSExecute/PSExecuteConnection.cs [hunk 1]
```diff
@@ -134,12 +159,17 @@ private QvxTable GetData(ScriptCode script, string username, string password, st
...
+                    else
+                    {
+                        // Wait for the Job to finish
+                        powerShell.AddCommand("Wait-Job");
+                        // Give all results and not the jobs back
+                        powerShell.AddCommand("Receive-Job");
+                    }
 
-                    using (var windowsGrandAccess = new WindowsGrandAccess(accountInfo,
+                    using (var windowsGrandAccess = new WindowsGrandAccess(accountInfo, useRemote,
```
Comment (konne) [inline src/QlikConnectorPSExecute/PSExecuteConnection.cs]: no way, WindowsGrantAccess has nothing todo with REMOTE, if accountInfo == null just du nothing

Comment (konne) [inline src/QlikConnectorPSExecute/PSExecuteConnection.cs]: WindowsGrantAccess(useRemote ? null : accountInfo,...)

File: src/QlikConnectorPSExecute/web/connectdialog.ts [hunk 2]
```diff
@@ -43,12 +46,19 @@ class ConnectDialog {
     }
 
     constructor(input: any, scope: any) {
+        console.log("input", input);
```
Comment (konne) [inline src/QlikConnectorPSExecute/web/connectdialog.ts]: remove this, it is not for production

File: src/QlikConnectorPSExecute/PSExecuteConnection.cs [hunk 3]
```diff
@@ -127,6 +152,9 @@ private QvxTable GetData(ScriptCode script, string username, string password, st
                         Array.ForEach(password.ToArray(), secPass.AppendChar);
                         powerShell.AddParameter("Credential", new PSCredential(username, secPass));
                         accountInfo = new NTAccount(username);
+
+                        if (useRemote == false && !UserSystemCheck(accountInfo))
```
Comment (konne) [inline src/QlikConnectorPSExecute/PSExecuteConnection.cs]: please change to !useRemote please get rid of the == true / == false

**Pull request merged**